app/views/offers/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Editing the offer</h1>
+
+<%= render 'form', offer: @offer %>
+
+<%= link_to 'Show', @offer %> |
+<%= link_to 'Back', root_path %>
```

Comment:
We probably don't need this since the Navbar already has a Home link.

Reply:
Should I then delete the show and back links or what do you mean by "this"?

Reply:
:) I mean the 'Back' link. I'm note sure what the show link does. It does not have a path specified?

Reply:
Regarding the 'Bank' link: I think if you just puts the instance, it automatically goes to the show page of the current one. So this should be fine.

https://www.rubyguides.com/2019/05/rails-link_to-method/
This part: <%= link_to "Improve Your Ruby Skills", @book %>